Users/Install.rst

```diff
@@ -60,6 +60,9 @@ system-wide, use:
     For this and all future steps, some steps require root access
     (also known as administrative privileges in Windows).
 
+    **Ubuntu Users** - Don't use ``apt-get install coala``
+    for installing coala as that is different software.
```

Comment:
Wrong place to insert this note as it breaks the context b/w the above and below lines :sweat_smile: